### PR TITLE
Some grammar changes by a native English speaker

### PR DESCRIPTION
**Pull request by phowell**
Hi, I saw the 0.9 announcement, and figured now was as good a time as any to finally learn specs. The original Book PR mentions that the author is not a native English speaker, and that help is welcome. So I figured I'd take a stab at it while I go through the tutorial. Here is my initial pass on the first two chapters. I also noticed that the code at the end of chapter 2 fails to compile so I made a couple of tweaks to fix that. Please let me know if there are any changes you would like me to make. If this is well received, I will try to find some time to work on the other chapters.

### CODE REVIEW

File: book/src/02_hello_world.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ impl<'a> System<'a> for HelloWorld {
 }
 ```
 
-This is how a system looks like. That is, a system which
-does not do anything (yet). Let's talk about the above dummy
-implementation first. The `SystemData` is an associated type
+This is what a system looks like. though it doesn't do anything (yet).
```
Comment (phowell) [inline book/src/02_hello_world.md]: Fixed. Oops. Somehow took two commits change a single letter.

**Comment by torkleyy** [comment]
Thanks for addressing my concerns!
Last thing: could you please squash the commits?

Since you seem to be fairly new to GH, here's how to do it:

* Ensure everything is either commited or stashed
* Type `git rebase -i HEAD~[NUM_COMMITS]`. This should give you a list of the last NUM_COMMITS commits.
* Change the "pick" in front of the last two commits to "s" for squash. This will squash them into the previous commits.
* You'll be asked to enter a combined commit message after that.
* Because your history cannot be merged into your fork, you need to do a force-push (`git push -f`) instead.

**Comment by phowell** [comment]
I think I fixed it, but accidentally included some old commits from you guys. If you are able to fix this, be my guest. If not, this serves me right for trying to figure this out while visiting family :(

**Comment by torkleyy** [comment]
There's no rush, so I'd recommend you try it yourself so you know how to do it. Here's how I'd do it (dunno if there's a better way):

Use `git log` to get the checksum of the latest commit (the one you want to push). Then, create a new branch based on `upstream/master` (`git checkout -b master upstream/master`).

You can now use `git cherry-pick <checksum>` to copy that commit over, then force-push and it should be done :wink: 

Thanks for your efforts!

**Comment by vitvakatu** [comment]
By the way, it's common practice to work not in `master` branch, but in local branch (for example `my_fix`, with command `git checkout master & git checkout -b my_fix`).
Then you'd add some commits to your `my_fix` branch.
Finally, you can create a PR based on this branch. It makes it much simpler to update your solution to the current master (in case you're working on issue for a long time) and gives you more flexibility.

It's just short notice, thank you for your first PR!

@torkleyy *nice solution, imo. I thought about creating new local branch, cherry-picking and creating new PR*

**Comment by phowell** [comment]
Okay, I think I finally got everything fixed. Yeah, I'm familiar with working on branches. It just slipped my mind at the time... and hey, it's just some simple text changes... what could possibly go wrong ;)

Anyway, here you go.

**Comment by kvark** [comment]
Weird, GH doesn't see a conflict.
bors retry

**Comment by torkleyy** [comment]
@kvark I think you have to put it into a separate line like this:

bors retry

**Comment by torkleyy** [comment]
Huh? Seems it doesn't work. cc @notriddle 

bors r+

**Comment by notriddle** [comment]
The retry thing is a known behavioral difference between bors-ng and homu. I never really saw the point of homu's way of doing it; it's one more command for newbies to learn for no apparent reason.

As for the merge conflict, congrats! You've found a bug! Apparently, because of the way you closed the PR, pushed to it, then reopened it, bors thinks that the commit hash for PR #198 is 52a8dec635fb68057dde5cf73611c1b16af5ff36, which is the same as master's current commit hash. This is wrong; it had that commit hash when it was closed, but you reopened the PR with commit hash fe48f66af24739f841b459664ba15da23b0f4517.

I've filed and issue, but you can work around it right now by going into the dashboard, opening up this repo, and clicking the Synchronize button.

**Comment by torkleyy** [comment]
Okay let's try it again :)

bors r+

**Comment by phowell** [comment]
Well, this has definitely been an educational experience. Thank you all for your patience. As promised, I'll try to take a look at the next few chapters some time this week.... and this time hopefully things will go smoother ;)

**Comment by torkleyy** [comment]
@phowell Great thanks again!

@notriddle Cheers for the great support!

**Pull request merged**